### PR TITLE
Acceptance tests support without vertica.

### PR DESCRIPTION
**Pull request by HassanJaveed84**
@mulby @brianhw 
- Fixed financial reports acceptance test(when vertica is not available).
- Skip internal reporting user_course test when vertica is not available.

I've started a run here:
http://jenkins-ci.analytics.edx.org/view/ad-hoc/job/edx-analytics-pipeline-acceptance-test-manual/805

### CODE REVIEW

File: edx/analytics/tasks/tests/acceptance/test_financial_reports.py [hunk 1]
```diff
@@ -96,14 +97,12 @@ def test_end_to_end_without_vertica(self):
...
+        output_targets = PathSetTask([output_root], ['*']).output()
         raw_output = ""
-        for file_name in files:
-            if file_name.startswith(target.path):
-                file_name = file_name[len(target.path) + 1:]
-            if file_name[0] != '_':
-                raw_output += get_target_from_url(url_path_join(output_root, file_name)).open('r').read()
+        for output_target in output_targets:
+            if output_target.path.startswith('s3://'):
+                output_target = get_target_from_url(output_target.path.replace('s3://', 's3+https://'))
```
Comment (mulby) [inline edx/analytics/tasks/tests/acceptance/test_financial_reports.py]: I think we could replace both of these lines with simply:

``` python
output_target = get_target_from_url(get_jenkins_safe_url(output_target.path))
```

Using this function:
https://github.com/edx/edx-analytics-pipeline/blob/hassan/acceptance-tests-support/edx/analytics/tasks/tests/acceptance/__init__.py#L94

**Comment by mulby** [comment]
FYI @bradenmacdonald - we are tweaking your test a bit to support running it on S3 (instead of HDFS)

**Comment by mulby** [comment]
@HassanJaveed84 - One small nit, overall :+1:

**Comment by bradenmacdonald** [comment]
@mulby Can it still run locally on a devstack?

**Comment by mulby** [comment]
@bradenmacdonald - It should still work, @HassanJaveed84 did you test this on your devstack?

**Comment by bradenmacdonald** [comment]
BTW, if the test is working well, you may want to run it unconditionally, and not just `@when_vertica_not_available` since I think it's a more comprehensive test than the vertica-based one.

**Comment by HassanJaveed84** [comment]
Wasn't able to test it on devstack due to the hive issue I mentioned. 

Tested it on Jenkins:
http://jenkins-ci.analytics.edx.org/view/ad-hoc/job/edx-analytics-pipeline-acceptance-test-manual/802/

**Comment by HassanJaveed84** [comment]
Should have mentioned, the change should still work for hdfs(devstack).

**Pull request merged**